key-derivation.md

```diff
@@ -46,7 +54,7 @@ credentialId = version || uniqueId || extState || credentialMac
 To support an optional deterministic mode, in which an observer that knows `seekKey` can verify that the authenticator generated the `credentialId` correctly, use the following formula for the `uniqueID`:
 
 ```
-uniqueId = SHA256HMAC(seedKey, b"uniqueId" || rpId || userId || hash)
+uniqueId = SHA256HMAC(seedKey XOR uniqueIdMask, rpId || userId || hash)
```

Comment:
Where does this completely new construction come from? I'd need some motivation.